src/test/kotlin/io/gitlab/arturbosch/detekt/idea/DetektStub.kt

```diff
@@ -20,7 +20,7 @@ import io.gitlab.arturbosch.detekt.api.Severity
 import io.gitlab.arturbosch.detekt.api.SourceLocation
 import io.gitlab.arturbosch.detekt.api.TextLocation
 import org.jetbrains.kotlin.com.intellij.openapi.util.Key
-import org.jetbrains.kotlin.psi.KtFile
+import detekt.shadow.org.jetbrains.kotlin.psi.KtFile
```

Comment:
note: need to use shadow version, this reference is unresolved in IDE, but `./gradlew check` is succeeded.